### DISCUSSION TITLE
GitHub copilot requires subscription for verified student in VSCode

### DISCUSSION THREAD

**Discussion by JuCarv-bit** [community/community#20129] [Copilot Conversations] [answered]
Hello,

As a verified student I'm eligible to use GitHub Copilot for free, but copilot has stopped showing code suggestions for a while -- I believe it is happening since when I start using the student pack. Instead, when checking the extension homepage on VSCode, I see a "Subscription required" button in orange in the "details" tab. Clicking the button, I'm redirected to the billing page: https://github.com/settings/billing

I do not know what is causing this problem, since I'm a 2022 student and according to [this thread ](https://github.com/orgs/github-community/discussions/19050) it should be working now.

Actions I tried:
- Updated VScode to the latest version
- Reinstalled the extension
- Sign out and Sing in with my GitHub account in VSCode

I'm using MacOs 11.6.1 if it can help

Looking forward to hearing from you. Thanks!

**Comment by johncorderox** [comment, marked as answer]
Have you tried checking to see if the JSON value is set to true for suggestions? 

Within the copilot suggestions settings, look for "Inline Suggest:Enable", and select or deselect the checkbox to enable or disable inline suggestions.

**Reply by JuCarv-bit** [reply]
Yes, inline suggest is enabled in copilot extension settings

<img width="909" alt="Screen Shot 2022-07-07 at 1 11 53 PM" src="https://user-images.githubusercontent.com/79609541/177821178-293c5bf4-5bf2-4d5a-8ec9-a3cd353e329e.png">

**Reply by JuCarv-bit** [reply]
Update: I proceed to uncheck and check again this box and restart VSCode and now my copilot is making suggestions. Thanks!!!
